lib/rules/declaration-property-value-keyword-no-deprecated/index.mjs

```diff
@@ -91,6 +91,7 @@ const PROPERTY_NAME_TO_KEYWORDS = new Map([
 	['zoom', { reset: '1' }],
 	['border-color', DEPRECATED_COLORS],
 	['scrollbar-color', DEPRECATED_COLORS],
+	['text-decoration', ['blink']],
 	['text-decoration-line', ['blink']],
```

Comment:
I don't know which browser support `text-decoration-line` **and** does not disable `blink` effect.
It's probably one or the other but to err on the side of caution I won't remove line 95.